cloud/ovh/ovh_ip_loadbalancing_backend.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+DOCUMENTATION = '''
+---
+module: ovh_ip_loadbalancing_backend
+short_description: Manage OVH IP LoadBalancing backends
+description:
+    - Manage OVH (French European hosting provider) LoadBalancing IP backends
+version_added: "1.9"
+author: Pascal HERAUD @pascalheraud
+notes:
+    - Uses the python OVH Api U(https://github.com/ovh/python-ovh). You have to create an application (a key and secret) with a consummer key as described into U(https://eu.api.ovh.com/g934.first_step_with_api)
+requirements: 
+    - ovh >  0.35
+options:
+    name:
+        required: true
+        description:
+            - Name of the LoadBalancing internal name (ip-X.X.X.X)
+    backend:
+        required: true
+        description:
+            - The IP address of the backend to update / modify / delete
+    state:
+        required: false
+        default: present
+        choices: ['present', 'absent']
+        description:
+            - Determines wether the backend is to be created/modified or deleted
+    probe:
+        required: false
+        default: none
+        choices: ['none', 'http', 'icmp' , 'oco']
+        description:
+            - Determines the type of probe to use for this backend
+    weight:
+        required: false
+        default: 8
+        description:
+            - Determines the weight for this backend
+    endpoint:
+        required: true
+        description:
+            - The endpoint to use ( for instance ovh-eu)
+    application_key:
+        required: true
+        description:
+            - The applicationKey to use
+    application_secret:
+        required: true
+        description:
+            - The application secret to use
+    consumer_key:
+        required: true
+        description:
+            - The consumer key to use
+    
+'''
+
+EXAMPLES = '''
+# Adds or modify a backend to a loadbalancing
+- ovh_ip_loadbalancing name=ip-1.1.1.1 ip=212.1.1.1 state=present probe=none weight=8 endpoint=ovh-eu application_key=yourkey application_secret=yoursecret consumer_key=yourconsumerkey
+
+# Removes a backend from a loadbalancing
+- ovh_ip_loadbalancing name=ip-1.1.1.1 ip=212.1.1.1 state=absent endpoint=ovh-eu application_key=yourkey application_secret=yoursecret consumer_key=yourconsumerkey
+'''
+
+import sys
+import ovh
```

Comment:
you should wrap that import in a try/catch ( like https://github.com/ansible/ansible-modules-extras/blob/devel/system/firewalld.py#L94 ) and show a error message if the ovh module is not present.